pages/1.11/installing/production/patching/index.md

```diff
@@ -35,7 +35,7 @@ Example: DC/OS 1.X.A to 1.X.B (1.11.1 --> 1.11.2)
 - DC/OS Enterprise downloads can be found [here](https://support.mesosphere.com/hc/en-us/articles/213198586-Mesosphere-Enterprise-DC-OS-Downloads). [enterprise type="inline" size="small" /]
 
 ## Supported patch paths
-- From the latest GA version of previous to the latest GA version of current. For example, if 1.8.8 is the latest and 1.9.0 is the latest, this patch would be supported.
+- From the latest GA version of previous to the latest GA version of current. For example: if 1.8.7 is the latest GA version of previous and 1.8.8 is the latest GA version of current then this patch would be supported.
```

Comment:
This bullet should be deleted.   This wording doesn't make any sense to me, and is redundant with the next bullet point. This bullet is left over from when 'patching' and 'upgrading' were all part of the same document; this bullet documented the supported upgrade paths   (i.e. from latest GA of previous release to the latest GA of the current release, like 1.9.10->1.10.8).   But that scenario is now considered an 'upgrade' and so shouldn't even be mentioned on this page.